.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,17 @@
-## This PR is a:
+## Purpose of this PR (required)
 
-- [ ] New topic
-- [ ] Content update
-- [ ] Content fix or rewrite
-- [ ] Bug fix or improvement
+Describe the goal and the types of changes this PR covers.
```

Comment:
```suggestion
Describe the goal and the type of changes this PR covers.
```

Reply:
Add a list of types.

It should correspond to the labels we use. The labels already contain short descriptions, so you could just copy the label description.

Reply:
Good idea.